### PR TITLE
Fixed Client/Server

### PR DESCRIPTION
**Pull request by wobbethan**
Restructure Package/Try-catch blocks

### CODE REVIEW

File: src/Threads/Client.java [hunk 1]
```diff
@@ -12,10 +12,10 @@ public class Client extends Thread {
     String message;
     String MESSAGE;
```
Comment (RyanMcPherson7) [inline src/Threads/Client.java:13]: why are there 2 `message` properties?

Comment (wobbethan) [reply src/Threads/Client.java:13]: Left over from provided sample, I don't think we actually wrote anything original for this yet.

Comment (wobbethan) [reply src/Threads/Client.java:13]: message is sent to server and MESSAGE is the response from server

Comment (RyanMcPherson7) [reply src/Threads/Client.java:13]: gotcha, we can remove this in a later commit

**Review by RyanMcPherson7 — APPROVED** [review]

**Pull request merged**